Polygon_mesh_processing/examples/Polygon_mesh_processing/remove_degeneracies_example_OM.cpp

```diff
@@ -20,7 +20,7 @@ int main(int argc, char* argv[])
   const char* filename = (argc > 1) ? argv[1] : "data/degtri_sliding.off";
 
   Mesh mesh;
-  if (!OpenMesh::IO::read_mesh(mesh, filename) || mesh.is_empty()) {
+  if (!OpenMesh::IO::read_mesh(mesh, filename) || num_vertices(mesh)) {
```

Comment:
The change is strange: `num_vertices(mesh)` evaluate to `true` if the mesh is **non-empty**, and that is the opposite of `mesh.is_empty()`.